taxonomies/ingredients.txt

```diff
@@ -4479,6 +4479,16 @@ ciqual_food_name:fr:Yaourt à la grecque, nature
 #ciqual_food_name:en:Yogurt, Greek-style, ewe's milk
 #ciqual_food_name:fr:Yaourt à la grecque, au lait de brebis
 
+<en:dairy
+en:skyr
```

Comment:
we can just have this:

en:skyr (we want a en: entry as the main name of all ingredients)
xx:skyr (will match skyr in all languages, and will be the default translation for all languages)

Reply:
Ah true I forgot, I modified it